sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-decisions.dmn

```diff
@@ -454,14 +454,14 @@
           <text>calculatedDataTotalEligibleDependants</text>
         </inputExpression>
       </input>
-      <input id="InputClause_070kjf4" label="studentDataLivingWithParents">
+      <input id="InputClause_070kjf4" label="studentDataLivingAtHome">
         <inputExpression id="LiteralExpression_04z1pfi" typeRef="string">
-          <text>studentDataLivingWithParents</text>
```

Comment:
Not a part of the PR. Considering that `studentDataLivingWithParents` is obsolete, we have an existing mapping from for the property `livingWithParents`  in workflow data for assessment gateway(and v2). 
![image](https://github.com/user-attachments/assets/97919887-c4df-43e7-bcff-d50a63d2c249)